### PR TITLE
[RFR] Add about modal widget and test page

### PR DESCRIPTION
**Pull request by mshriver**
In progress about modal and a test page for it

### CODE REVIEW

File: testing/testing_page.html [hunk 1]
```diff
@@ -34,5 +34,35 @@
...
+          <h1>Widgetastic About Modal</h1>
+          <div class="product-versions-pf">
+            <ul class="list-unstyled">
+              <li><strong>Field1</strong> Field1Value</li>
+              <li><strong>Field2</strong> Field2Value</li>
+              <li><strong>Field3</strong> Field3Value</li>
+            </ul>
+          </div>
+          <div class="trademark-pf">
+            Widgetsatic Trademark and Copyright Information
```
Comment (mfalesni) [inline testing/testing_page.html]: typo.

File: src/widgetastic_patternfly/__init__.py [hunk 2]
```diff
@@ -1380,3 +1380,58 @@ def __init__(self, parent, id=None, name=None, logger=None):
...
+class AboutModal(Widget):
+    """
+    Represents the patternfly about modal
+
+    Provides a close method
+    """
+    ROOT = ParametrizedLocator('//div[normalize-space(@id)={@id|quote} and '
+                               'contains(@class, "modal") and contains(@class, "fade")]')
+    CLOSE_LOC = './/div[@class="modal-header"]/button[@class="close" and @data-dismiss="modal"]'
+    TITLE_LOC = './/div[@class="modal-body"]/h2'
```
Comment (mfalesni) [inline src/widgetastic_patternfly/__init__.py]: I believe if you define this as a Text widget, it should work as well. Similar with some of the other locators.

File: testing/test_about_modal.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        button = Button('Launch about modal')
+        about = AboutModal(id=modal_id)
+
+    view = TestView(browser)
+    assert not view.about.is_open
+
+    # Open the modal
+    assert view.button.title == 'Launch Modal'
+    assert not view.button.disabled
+    view.button.click()
```
Comment (mshriver) [inline testing/test_about_modal.py]: This doesn't seem to be working in PhantomJS so travis won't pass.

File: src/widgetastic_patternfly/__init__.py (hunk 2 above)
Comment (mshriver) [inline src/widgetastic_patternfly/__init__.py]: Asked about this offline as well.

I'm not following the implementation you're describing here.  Tried defining a `self.title_widget` within the constructor using a Text widget, but then just have a widget_descriptor when I try to access it in the property.

Not seeing any other locators that are implemented as Text widgets?

Comment (mfalesni) [inline src/widgetastic_patternfly/__init__.py]: Widgets can only be defined on the class, not in constructor.

**Pull request merged**